extension/ballastextension/memory_ballast.go

```diff
@@ -25,17 +25,32 @@ import (
 const megaBytes = 1024 * 1024
 
 type memoryBallast struct {
-	cfg     *Config
-	logger  *zap.Logger
-	ballast []byte
+	cfg       *Config
+	logger    *zap.Logger
+	ballast   []byte
+	memGetter func() (uint64, error)
```

Comment:
I think it makes sense to name this sort of func as a func name unless there's some go style I don't know about. E.g. `getTotalMem`